stays/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length = 45)
+
+    class Meta:
+        db_table = 'categories'
+
+class Staytype(models.Model):
+    category    = models.ForeignKey('Category', on_delete = models.PROTECT)
+    name        = models.CharField(max_length = 45)
+    address     = models.CharField(max_length = 2000)
+    longitude   = models.DecimalField(max_digits = 10, decimal_places = 7)
+    latitude    = models.DecimalField(max_digits = 10, decimal_places = 7)
+    description = models.TextField()
+    facility    = models.ManyToManyField('Facility', through = 'StaytypeFacility', related_name = 'staytypes')
+
+    class Meta:
+        db_table = 'staytypes'
+
+class StaytypeFacility(models.Model):
+    staytype  = models.ForeignKey('Staytype', on_delete = models.CASCADE)
+    facility  = models.ForeignKey('Facility', on_delete = models.CASCADE)
+
+    class Meta:
+        db_table = 'staytypes_facilities'
+
+class Facility(models.Model):
+    name = models.CharField(max_length = 45)
+
+    class Meta:
+        db_table = 'facilities'
+
+class StaytypeImage(models.Model):
+    image_url = models.CharField(max_length = 2000)
+    staytype  = models.ForeignKey('Staytype', on_delete = models.CASCADE)
+
+    class Meta:
+        db_table = 'staytypeimages'
+
+
+class Room(models.Model):
+    staytype  = models.ForeignKey('Staytype', on_delete = models.PROTECT)
+    name      = models.CharField(max_length = 45)
+    quantity  = models.IntegerField()
+    image_url = models.CharField(max_length = 2000)
+    people    = models.PositiveIntegerField()
+
+    class Meta:
+        db_table = 'rooms'
+
+class RoomOption(models.Model):
+    room      = models.ForeignKey('Room', on_delete = models.PROTECT)
+    name      = models.CharField(max_length = 45)
+    price     = models.DecimalField(max_digits = 10, decimal_places = 2)
+    check_in  = models.TimeField()
+    check_out = models.TimeField()
```

Comment:
TimeField로 날짜 시간 전부 저장할 수 있나요??

```suggestion
    check_in  = models.TimeField()
    check_out = models.TimeField()
```

Reply:
객실페이지에 대한 GET 요청시  각 객실의 체크인 시간과 체크아웃 시간을 위해 작성한 column입니다. 아래와 같이 응답하고자, 날짜가 필요없다고 생각되어 TimeField로만 지정하였습니다.  

![스크린샷 2021-08-18 오후 7 33 19](https://user-images.githubusercontent.com/63541271/129883690-c051b8c3-8696-41cb-b1c9-47d6b9f5c465.png)